src/app/containers/Chat/index.tsx

```diff
@@ -17,13 +18,21 @@ const onChatSelect = async (currentId: number, nextId: number) => {
   }
 };
 
-const MessageItem = ({ id, from_id, date, message }: MtpMessage) =>
+const MessageItem = ({ id, from_id, date, message, entities }: MtpMessage) =>
   <Message
     key={id}
     id={id}
     date={date}
     user={from_id}
-    text={message} />;
+    // HACK: what to do here?
+    text={entities
+      // TODO: remove any cast
+      // tslint:disable-next-line
+      ? <RichText
```

Comment:
Proposal: there should be only RichText in every case, `defaultProps = { entities: [] }`

But it's not important right now